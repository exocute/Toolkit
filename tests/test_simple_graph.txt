Graph com.exocute.pipeline.BranchingPipeline
	Activity a com.batatas