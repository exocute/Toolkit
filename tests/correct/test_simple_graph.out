Name: com.exocute.pipeline.BranchingPipeline
Id: a, Name: com.batatas
